### PR TITLE
Fix Osmosis - Apollo Vaults

### PR DESCRIPTION
**Pull request by phamminh0811**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/apollodao exports TVL: 
        
 
 
    terra                     0
    osmosis                   31.91 k
    
    total                    31.91 k

File: projects/helper/chain/cosmos.js [hunk 1]
```diff
@@ -203,6 +203,53 @@ async function sumTokens({ balances = {}, owners = [], chain, owner }) {
   return transformBalances(chain, balances);
 }
 
+async function tvlFromLockedGammTokens(contractAddresses, chain) {
```
Comment (g1nt0ki) [inline projects/helper/chain/cosmos.js:206]: I know it is still in draft mode, but if this code is linked with only one project/adapter, I dont think it should be common helpers file

**Comment by llamatester** [comment]
Error while running adapter at projects/apollodao: 
        
 
  
    
    
    ReferenceError: axios is not defined
    Truncated error stack:
    
        at tvlFromLockedGammTokens (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/apollodao/index.js:36:26)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/apollodao/index.js:26:21)
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:29)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:148:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:136:29
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:131:44

**Comment by llamatester** [comment]
The adapter at projects/apollodao exports TVL: 
        
 
 
    terra                     0
    osmosis                   32.72 k
    
    total                    32.72 k

**Comment by llamatester** [comment]
The adapter at projects/apollodao exports TVL: 
        
 
 
    terra                     0
    osmosis                   32.63 k
    
    total                    32.63 k

**Comment by llamatester** [comment]
The adapter at projects/apollodao exports TVL: 
        
 
 
    terra                     0
    osmosis                   2.22 M
    
    total                    2.22 M

**Comment by llamatester** [comment]
The adapter at projects/apollodao exports TVL: 
        
 
 
    terra                     0
    osmosis                   2.30 M
    
    total                    2.30 M

**Comment by llamatester** [comment]
The adapter at projects/apollodao exports TVL: 
        
 
 
    terra                     0
    osmosis                   2.30 M
    
    total                    2.30 M

**Comment by llamatester** [comment]
The adapter at projects/apollodao exports TVL: 
        
 
 
    terra                     0
    osmosis                   2.30 M
    
    total                    2.30 M

**Comment by llamatester** [comment]
The adapter at projects/apollodao exports TVL: 
        
 
 
    terra                     0
    osmosis                   2.30 M
    
    total                    2.30 M

**Comment by llamatester** [comment]
The adapter at projects/curve exports TVL: 
        
 
 
    harmony                   0
    ethereum-staking          494.74 M
    celo                      22.62 M
    xdai                      16.92 M
    aurora                    17.30 k
    arbitrum                  80.90 M
    moonbeam                  528.12 k
    optimism                  40.48 M
    avax                      16.62 M
    kava                      14.33 M
    polygon                   67.24 M
    fantom                    20.06 M
    ethereum                  3.57 B
    staking                   494.74 M
    
    total                    3.85 B

**Comment by sunnya97** [comment]
Should probably include the relaunch on osmosis date

**Comment by g1nt0ki** [comment]
@phamminh0811 thanks for the PR, I squashed and included your changes, skipped the lint fix

@sunnya97 do you have a date? we can add it as an event to the timeline and probably try to refill tvl chart till then

**Comment by apollo-sturdy** [comment]
@g1nt0ki First vault on Osmosis was instantiated on Feb 16, 2023

**Pull request closed without merging**